weave/graph_client.py

```diff
@@ -74,6 +74,14 @@ def ref_input_to(self, ref: artifact_wandb.WandbArtifactRef) -> WeaveIter[Run]:
             filter_node = rows_node.filter(lambda row: row["inputs._ref0"] == ref)  # type: ignore
             return WeaveIter(filter_node, cls=Run)
 
+    def ref_value_input_to(
+        self, ref: artifact_wandb.WandbArtifactRef
+    ) -> WeaveIter[Run]:
+        with context_state.lazy_execution():
+            rows_node = self.runs_st.rows()
+            filter_node = rows_node.filter(lambda row: row["inputs._ref_digest0"] == ref.digest)  # type: ignore
```

Comment:
using `_ref_digest0` (and similarly `_ref0`) is a hack with the consequence that you can only have this for the 0th param. ideally we remove these enumerated attributes and make them lists.